users/signals.py

```diff
@@ -2,13 +2,53 @@
 
 from urllib.parse import urljoin
 
+from django.db.models.signals import post_save
 from django.dispatch import receiver
 from django.conf import settings
+from django.contrib.auth.models import User
 from django.core.mail import EmailMultiAlternatives
 from django.template.loader import render_to_string
 
 from django_rest_passwordreset.models import get_password_reset_token_expiry_time
 from django_rest_passwordreset.signals import reset_password_token_created
+from users.models import EmailConfirmationToken, Person
+
+
+@receiver(post_save, sender=User)
+def create_person(sender,  # pylint: disable=unused-argument
+                  instance, created, **_kwargs):
+    """Create a person and send email for confirmation user email. """
```

Comment:
Creates a Person instance and sends an email to the user, letting them to confirm their email address.